python/datasources/age_adjust.py

```diff
@@ -0,0 +1,204 @@
+import ingestion.standardized_columns as std_col
```

Comment:
Should this file name by more specific to the CDC data? What happens if we add age-adj to other data sources; will those calculations happen in this file or in their own version of this file?

Reply:
Yeah probably